### PR TITLE
fix: moving out of the ActionWrapper the error message

### LINKED ISSUE #5290: An error dialog that appears when image is pushed to unavailable cluster is shown only when image's actions context menu is opened
### Bug description

Expected error dialog is shown only withing context menu from image's actions.

pushimagerror_badly_shown

### Operating system

Windows, Linux

### Installation Method

Installer from website/GitHub releases

### Version

1.6.3

### Steps to reproduce

1. Pull an image
2. Push image to a Kind (for example) which does not exist
3. Assert: Error popup is shown
Actual result: Popup is part of context menu over image's actions

### Relevant log output

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by axel7083**
### What does this PR do?

Fix #5290

### What issues does this PR fix or reference?

Its moves the error body outside the ActionWrapper

### How to test this PR?

- Move an image to a non-existing or unavailable cluster

### CODE REVIEW

File: packages/renderer/src/lib/image/ImageActions.svelte [hunk 1]
```diff
@@ -96,6 +96,36 @@ function onError(error: string): void {
...
+    <div class="border-t-red-600 border-t-2 p-4 bg-charcoal-600" aria-label="Success alert">
+      <div class="flex flex-row justify-center items-center pb-2">
+        <Fa icon="{faExclamationCircle}" class="text-red-500 mr-2" />
+        <div class="text-red-500 font-bold text-sm">
+          {errorTitle}
+        </div>
+        <Fa
+          icon="{faTimes}"
+          class="text-gray-900 pl-2 cursor-pointer"
+          on:click="{() => {
```
Comment (lstocchi) [inline packages/renderer/src/lib/image/ImageActions.svelte:110]: Not related to this PR as it was already there but this doesn't work. The `on:click` on a Fa item is not triggered properly, at least on Windows. We should open an issue

**Review by lstocchi — APPROVED** [review]
LGTM

File: packages/renderer/src/lib/image/ImageActions.svelte (hunk 1 above)
Comment (axel7083) [reply packages/renderer/src/lib/image/ImageActions.svelte:110]: I can confirm the close icon does not work, I opened https://github.com/containers/podman-desktop/issues/5420

**Comment by axel7083** [comment]
Closing in favor of https://github.com/containers/podman-desktop/pull/5421

**Pull request closed without merging**